packages/devtools_app/integration_test/run_tests.dart

```diff
@@ -26,23 +26,41 @@ const _testDirectory = 'integration_test/test';
 const _testSuffix = '_test.dart';
 const _offlineIndicator = 'integration_test/test/offline';
 
+// A mapping of test app device to the unsupported tests for that device.
```

Comment:
nit: three / for dart doc

Reply:
Done, also moved to be a private static field of the `TestAppDevice` enhanced enum